Slider/Assets/Scripts/UI/Settings/SettingsManager.cs

```diff
@@ -98,4 +106,33 @@ public static Setting<T> Setting<T>(Settings setting)
     {
         return (Setting<T>)settings[setting];
     }
+
+    /// <summary>
+    /// Retrieves the Setting data class for the specified Setting. This can be used to access properties such as the CurrentValue, DefaultValue, etc.
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    /// <param name="setting"></param>
+    /// <returns></returns>
+    public static ISetting Setting(Settings setting)
+    {
+        return settings[setting];
+    }
+}
+
+[System.Serializable]
+public class Resolution
```

Comment:
Will move this out later when i finish up the graphics settings

Reply:
okay pookie